include-list.txt

```diff
@@ -0,0 +1,15 @@
+# This is a list of python additional packages needed for older versions of ESP-IDF (v4.3, v4.4). This file is used with pip.
```

Comment:
We might stress out that this list is not just for those cases. 

At some point we need to start to update the README as well. This file should be mentioned there. Or where to update the actively supported IDF versions. For this reasons, those variables might be placed in a separate file, e.g. `constants.py`.

This can be started now or left to the end.